.github/workflows/lint_python.yml

```diff
@@ -6,16 +6,17 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
-      - run: pip install --upgrade pip wheel
-      - run: pip install black flake8 flake8-2020 flake8-bugbear
-                         flake8-comprehensions mypy pytest pyupgrade safety
+        with:
+          cache: 'pip'
+      - run: |
+          pip install --upgrade pip wheel
+          pip install .[all]
       - run: black --check . || true
-      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88
-                      --show-source --statistics
-      - run: pip install -r pex-requirements.txt -r tests/requirements.txt
-      - run: mkdir --parents --verbose .mypy_cache
-      - run: mypy --ignore-missing-imports --install-types --non-interactive . || true
+      - run: flake8 . --max-complexity=43 --max-line-length=102 --show-source --statistics
+                      --ignore=B001,B006,B007,B011,B014,E121,E122,E128,E265,E712,E722,E741,F401,F811,F841,R502,R503,R504,W503
```

Comment:
```suggestion
      - run: flake8 .  # flake8 configuration can be found in setup.py
```

Reply:
Maybe: `s/found in setup.py/found in setup.cfg/` ?

Reply:
2c1337b